docs/dev/troubleshooting/withdrawal-delay.md

```diff
@@ -1,12 +1,65 @@
+
+1. **Delay in block confirmation**: as mentioned in the [withdrawal delay](#withdrawal-delay) section above, successful withdrawals may be subject to an ongoing confirmation process. Consequently, you won't be able to see your transaction on the block explorer until the confirmation process completes.
+
+2. **Transaction reverted**: a withdrawal may fail to appear on the block explorer if the transaction is reverted due to a conflict or issue with the smart contract that executed the transaction. In such cases, the transaction is canceled and it won't be included on the block explorer.
+
+3. **Wrong address**: If a user mistakenly sends tokens to an incorrect address, the transaction will not be visible on the block explorer, and the tokens will not arrive in the intended recipient's wallet.
+
+### Transactions in Etherscan
+
+The **Transactions** section in Etherscan displays transactions between two Ethereum addresses. Each transaction has details such as the transaction hash, the block number, the timestamp, the sender and receiver addresses, the amount of Ether or tokens involved, and the transaction fee. These transactions are confirmed by the network and are typically sent from one external address to another.
+
+On the other hand, **Internal Transactions** are initiated by a smart contract or other internal code execution within the Ethereum network. Although these transactions can be prompted by user activity, they are not sent directly from one address to another but are instead part of the internal workings of a smart contract. Internal transactions may involve the transfer of ETH or ERC20 tokens between different addresses within the contract. 
+
+::: tip
+Withdrawals from the zkSync Era network are typically internal transactions managed by the [zkSync Era Diamond Proxy](https://etherscan.io/address/0x32400084c286cf3e17e7b677ea9583e60a000324) contract. These transactions are recorded in the **Internal Transactions** section in Etherscan due to their internal nature.
+:::
+
+In summary, the **Transactions** section in Etherscan displays transactions between external addresses, while the **Internal Transactions** section displays transactions that occur within smart contracts.
+
+### How to check your internal transactions
+
+To check the internal transactions of an Ethereum wallet, follow these steps:
+
+
+1. Launch your web browser and visit [Etherscan.io](https://etherscan.io/).
+
+2. In the search bar at the top of the page, enter the Ethereum wallet address you used to withdraw funds. This should be the address you withdrew funds from, not the destination address.
+
+![Etherscan search bar](../../assets/images/ether-search.png "Search for transactions")
+
+3. Click **Enter**.
+
+4. Scroll down to the **Internal Transactions** section.
+
+Refer to the image below for more clarity.
```

Comment:
```suggestion
```

Reply:
This is unnecessary, unless there is something weird about the image that means the user wouldn't look at it automatically.